distribution/lib/Standard/Database/0.0.0-dev/src/Internal/SQLite/SQLite_Type_Mapping.enso

```diff
@@ -0,0 +1,180 @@
+from Standard.Base import all
+import Standard.Base.Errors.Illegal_Argument.Illegal_Argument
+import Standard.Base.Errors.Illegal_State.Illegal_State
+from Standard.Base.Function import const
```

Comment:
I think this should be in `from Standard.Base import all`

Reply:
The line `from Standard.Base.Function import const` seems unnecessary indeed. Removing.